lib/publiccloud/provider.pm

```diff
@@ -444,18 +447,25 @@ sub terraform_destroy {
     my $cmd;
     return if get_required_var('TEST') =~ /^publiccloud_upload_img/;
     record_info('INFO', 'Removing terraform plan...');
-    if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
+    f (get_var('PUBLIC_CLOUD_SLES4SAP')) {
```

Comment:
looks like a ooops

Reply:
Yes, bad copy/paste... thx!

Reply:
And can we make all "SLES4SAP" conditions within this single if.. so afterwards we just have a `$destroy_cmd` which we will use.

Reply:
Yes, will do the change.

Reply:
thx a lot

Reply:
Done, let me know if it's ok for you :-)